const t=JSON.parse(`{"key":"v-74875426","path":"/%E5%BC%80%E5%8F%91%E6%97%A5%E5%BF%97/JS/%E5%85%B3%E4%BA%8Easync%E5%92%8Cawait.html","title":"关于async和await","lang":"zh-CN","frontmatter":{"title":"关于async和await","category":["开发日志","JS"],"description":"发现问题 某天在牛客上刷到这样一道题： 问输出结果是什么？我想这也太简单了，肯定是返回一个成功状态的 Promise 啊，一看答案我傻了，居然是一个未解决的 Promise. 最后我好好想了想，之前的理解一直是错的。看了几篇大牛的文章，我学废了 问题分析 看回上面那段代码，async 函数返回一个 Promise 对象，可以使用 then 方法添加回调...","head":[["meta",{"property":"og:url","content":"https://zfhblog.top/%E5%BC%80%E5%8F%91%E6%97%A5%E5%BF%97/JS/%E5%85%B3%E4%BA%8Easync%E5%92%8Cawait"}],["meta",{"property":"og:site_name","content":"ZFH's Blog"}],["meta",{"property":"og:title","content":"关于async和await"}],["meta",{"property":"og:description","content":"发现问题 某天在牛客上刷到这样一道题： 问输出结果是什么？我想这也太简单了，肯定是返回一个成功状态的 Promise 啊，一看答案我傻了，居然是一个未解决的 Promise. 最后我好好想了想，之前的理解一直是错的。看了几篇大牛的文章，我学废了 问题分析 看回上面那段代码，async 函数返回一个 Promise 对象，可以使用 then 方法添加回调..."}],["meta",{"property":"og:type","content":"article"}],["meta",{"property":"og:locale","content":"zh-CN"}],["meta",{"property":"og:updated_time","content":"2023-01-07T02:29:19.000Z"}],["meta",{"property":"article:modified_time","content":"2023-01-07T02:29:19.000Z"}],["script",{"type":"application/ld+json"},"{\\"@context\\":\\"https://schema.org\\",\\"@type\\":\\"Article\\",\\"headline\\":\\"关于async和await\\",\\"image\\":[\\"\\"],\\"dateModified\\":\\"2023-01-07T02:29:19.000Z\\",\\"author\\":[]}"]]},"headers":[{"level":2,"title":"发现问题","slug":"发现问题","link":"#发现问题","children":[]},{"level":2,"title":"问题分析","slug":"问题分析","link":"#问题分析","children":[]},{"level":2,"title":"备注","slug":"备注","link":"#备注","children":[]},{"level":2,"title":"参考文章","slug":"参考文章","link":"#参考文章","children":[]}],"git":{"createdTime":1670983849000,"updatedTime":1673058559000,"contributors":[{"name":"zhangfanhang","email":"543629463@qq.com","commits":2}]},"readingTime":{"minutes":3,"words":901},"filePathRelative":"开发日志/JS/关于async和await.md","localizedDate":"2022年12月14日","copyright":{},"autoDesc":true}`);export{t as data};
